doc/reports/spikes/ras-rm-account-removal.md

```diff
@@ -0,0 +1,56 @@
+We need to look into the different ways we can remove an account if a request is made for us to do so.
+There seem to be 2 ways to do this, these are
+  - Obfuscating the personal details of the account, but leaving the rest of it intact.
+  - Extracting the record of the account from the database, along with any other records that could/should be reasonably removed.
+
+Below I'll detail the rough steps involved for both appraoches, as well as any other considerations that need to be made
+
+# Obfuscating personal details
```

Comment:
I updated this to be more precise with my words.  Also added a step to disable enrolment and a consideration about HOW we should do this.  The coding for the obfuscated email address is almost certainly going to lead to some code that you'll need esoteric knowledge of to be able to maintain it.  Making enrolment disabling mandatory is probably the safest option.

Reply:
Will obfuscating the data result in the counts being wrong for the emails expected etc? As the business rely on these numbers, could these appear in the response chasing file and the business thinking that because there's an email address then an email needs to be sent or should be sent if the account has not been unenrolled from surveys? Have we also thought about the complication of GDPR with regards to this? I think we need to speak to IA with regards to GDPR and this.

Reply:
Did we get an answer to this?